web/src/model/topology.ts

```diff
@@ -34,5 +53,245 @@ export const DefaultOptions: TopologyOptions = {
   edgeTags: true,
   maxEdgeValue: 0,
   startCollapsed: false,
-  groupTypes: TopologyGroupTypes.NAMESPACES
+  truncateLabels: true,
+  groupTypes: TopologyGroupTypes.NAMESPACES,
+  lowScale: 0.3,
+  medScale: 0.5
+};
+
+export const DEFAULT_NODE_TRUNCATE_LENGTH = 25;
+export const DEFAULT_NODE_SIZE = 75;
```

Comment:
this is nitpicking, but still it would be good to get a common approach on that: which naming case for constants?

I tend to follow the other convention, `defaultNodeSize`, making no difference between variables and constants, being exported or not, being global or not. I think it's the most common in jasvascript / typescript landscape (though I wouldn't bet on it), it's in the well adopted [basarat's TypeScript book](https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md#variable-and-function) or in the [TypeScript language convention](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines) itself (which does **not** pretend to be a convention for typescript projects in general)

On the other hand, we have the [google approach](https://google.github.io/styleguide/tsguide.html) which makes a difference between global constants and other constants, like you're doing here. The same approach can be seen[ elsewhere](https://ts.dev/style/#identifiers) (I don't think ts.dev has anything official with TypeScript, it's just a service company)

Again, this is nitpicking, but if we get a common approach it would be more consistent and we'd clear the current ambiguity that we have while writing code or doing reviews. I'm more used to doing the first option in typescript, but I'm fine to change

Reply:
It seems like the console code tend to follow capital-case approach, while not 100% consistent

Reply:
I'm ok with any. We can centralize all the exported constants to a single folder so it will be easier to maintain with a single convention ?

Reply:
I prefer the domain-driven approach than everything-centralized :) Keep the code closer to where it's used, and try to have some domain boundaries, rather than a single constants file that often grows out of control, can easily turn up being messy and sometimes ends up having unused constants that remain unnoticed.. I've been there :)

Reply:
I have opened a follow up task since this is not related to that PR
https://issues.redhat.com/browse/NETOBSERV-268
![image](https://user-images.githubusercontent.com/91894519/161962505-079d76fc-1e88-466e-89fe-9d4ac5a496df.png)